src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseEntitySelectPage.tsx

```diff
@@ -40,31 +42,45 @@ function QuickbooksOutOfPocketExpenseEntitySelectPage({policy}: WithPolicyConnec
                 keyForList: CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.CHECK,
                 isSelected: reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.CHECK,
                 isShown: !isLocationsEnabled,
+                accounts: bankAccounts || [],
             },
             {
                 value: CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY,
                 text: translate(`workspace.qbo.accounts.journal_entry`),
                 keyForList: CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY,
                 isSelected: reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY,
                 isShown: !isTaxesEnabled || isLocationsEnabled,
```

Comment:
```suggestion
                isShown: !isTaxesEnabled,
```

When `syncTax` is set to `true`, the Journal Entry option should not be displayed, even if Location import is enabled.

If we commit this change, however, there won't be any option displayed when both the tax import and location import are enabled. I think this is the expected behavior.

Reply:
[discussing about this in Slack](https://expensify.slack.com/archives/C036QM0SLJK/p1715013525305559?thread_ts=1714400674.229349&cid=C036QM0SLJK)

Reply:
[here is the new discussion](https://expensify.slack.com/archives/C036QM0SLJK/p1715186718773159?thread_ts=1714400674.229349&cid=C036QM0SLJK)

Reply:
Sounds unrelated to this PR.